pkg/deploy/controller/deploymentconfig/factory.go

```diff
@@ -53,13 +53,11 @@ func NewDeploymentConfigController(dcInformer, rcInformer, podInformer cache.Sha
 	})
 	c.rcStore.Indexer = rcInformer.GetIndexer()
 	rcInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc:    c.addReplicationController,
```

Comment:
Squash it to https://github.com/openshift/origin/pull/12143/commits/91756f5542b08d4bb64063d8df45aede38123f1e